SWLOR.Game.Server/Service/PlayerStatService.cs

```diff
@@ -473,21 +469,25 @@ public static EffectiveItemStats GetPlayerItemEffectiveStats(NWPlayer player, NW
 
 
                 // Calculate AC
-                if (ItemService.ArmorBaseItemTypes.Contains(item.BaseItemType) ||
-                    ItemService.ShieldBaseItemTypes.Contains(item.BaseItemType))
+                if (ItemService.ArmorBaseItemTypes.Contains(item.BaseItemType))
```

Comment:
Why are shields removed from this calculation?

Reply:
Basically it makes it so AC is not calculated on shields.  Fix not a change.

Reply:
Ok makes sense. I didn't realize they weren't supposed to grant AC at all :P